.babelrc

```diff
@@ -1,6 +1,22 @@
 {
   "presets": ["@babel/env", "@babel/react"],
   "plugins": [
+    [
+      "i18next-extract",
```

Comment:
This plugin scans through codebase and generates the translation files for all the supported languages!!

Reply:
So lets say if I add a new component somewhere `<Text>t('Vishal is awesome')</Text>`.

Plugin will append this key to every language file.